### DISCUSSION TITLE
AH00898: Error reading from remote server returned by /favicon.ico

### DISCUSSION THREAD

**Discussion by Gmello1** [netbox-community/netbox#12900] [Q&A] [answered]
Hey Everyone. I'm installing NETBOX in a RHEL 8 Server using apache HTTPD as reverse proxy but i cant connect to login page. In the web browser i keep getting a 502 Proxy Error. Searching for logs in **/etc/httpd/logs/error_log** i get the following log: **AH00898: Error reading from remote server returned by /favicon.ico**

All services are running properly, am i missing something? Thanks!

**Comment by candlerb** [comment]
> In the web browser i keep getting a 502 Proxy Error

That generally means that the Netbox service isn't running properly. Check the output of:

```
systemctl status netbox
journalctl -eu netbox
```

and check if there are any errors.  If it claims these are running, then check tcpdump for traffic between the Apache proxy and the Netbox gunicorn process:

```
tcpdump -i lo -nn -s0 -A tcp port 8001
```

(then try to access the web interface while this is running)

Don't worry too much about favicon.ico - it sounds like you have a more important issue to fix.

**Reply by Gmello1** [reply]
Thanks for you reply. 

See the output from journal: (i restarted netbox service to get updated data)


**Jun 14 15:15:47 sp3linnetbox systemd[1]: Stopping NetBox WSGI Service...
Jun 14 15:15:47 sp3linnetbox gunicorn[179429]: [2023-06-14 15:15:47 -0300] [179429] [INFO] Handling signal: term
Jun 14 15:16:17 sp3linnetbox gunicorn[179479]: [2023-06-14 18:16:17 +0000] [179479] [INFO] Worker exiting (pid: 179479)
Jun 14 15:16:17 sp3linnetbox gunicorn[179429]: [2023-06-14 15:16:17 -0300] [179429] [INFO] Shutting down: Master
Jun 14 15:16:17 sp3linnetbox gunicorn[179429]: [2023-06-14 15:16:17 -0300] [179429] [WARNING] Worker with pid 179479 was terminated due to signal 9
Jun 14 15:16:17 sp3linnetbox gunicorn[179429]: [2023-06-14 15:16:17 -0300] [179429] [WARNING] Worker with pid 179495 was terminated due to signal 9
Jun 14 15:16:17 sp3linnetbox gunicorn[179429]: [2023-06-14 15:16:17 -0300] [179429] [WARNING] Worker with pid 179585 was terminated due to signal 9
Jun 14 15:16:17 sp3linnetbox gunicorn[179429]: [2023-06-14 15:16:17 -0300] [179429] [WARNING] Worker with pid 179700 was terminated due to signal 9
Jun 14 15:16:17 sp3linnetbox gunicorn[179429]: [2023-06-14 15:16:17 -0300] [179429] [WARNING] Worker with pid 179745 was terminated due to signal 9
Jun 14 15:16:17 sp3linnetbox systemd[1]: netbox.service: Succeeded.
Jun 14 15:16:17 sp3linnetbox systemd[1]: Stopped NetBox WSGI Service.
Jun 14 15:16:17 sp3linnetbox systemd[1]: Started NetBox WSGI Service.
Jun 14 15:16:17 sp3linnetbox gunicorn[312181]: [2023-06-14 15:16:17 -0300] [312181] [INFO] Starting gunicorn 20.1.0
Jun 14 15:16:17 sp3linnetbox gunicorn[312181]: [2023-06-14 15:16:17 -0300] [312181] [INFO] Listening at: http://127.0.0.1:8001 (312181)
Jun 14 15:16:17 sp3linnetbox gunicorn[312181]: [2023-06-14 15:16:17 -0300] [312181] [INFO] Using worker: gthread
Jun 14 15:16:17 sp3linnetbox gunicorn[312264]: [2023-06-14 15:16:17 -0300] [312264] [INFO] Booting worker with pid: 312264
Jun 14 15:16:17 sp3linnetbox gunicorn[312265]: [2023-06-14 15:16:17 -0300] [312265] [INFO] Booting worker with pid: 312265
Jun 14 15:16:17 sp3linnetbox gunicorn[312373]: [2023-06-14 15:16:17 -0300] [312373] [INFO] Booting worker with pid: 312373
Jun 14 15:16:17 sp3linnetbox gunicorn[312374]: [2023-06-14 15:16:17 -0300] [312374] [INFO] Booting worker with pid: 312374
Jun 14 15:16:17 sp3linnetbox gunicorn[312474]: [2023-06-14 15:16:17 -0300] [312474] [INFO] Booting worker with pid: 312474**

tcp dump output:
18 packets captured
36 packets received by filter
0 packets dropped by kernel

**Reply by candlerb** [reply]
tcpdump is not showing any packets? That's weird: either the Apache proxy isn't sending the requests to Netbox at all, or you have some SELinux nonsense which is preventing tcpdump from writing to the console.

You could try putting SELinux into permissive mode, then re-running tcpdump.  If you still see nothing, you'll need to check the Apache config that it is really proxying to <http://127.0.0.1:8001/>

**Reply by Gmello1** [reply]
my apologies, i pasted only the summary, tcpdump is actually showing packages, see attached.
[tcpdump.txt](https://github.com/netbox-community/netbox/files/11750945/tcpdump.txt)

**Reply by candlerb** [reply, marked as answer]
Hmm... so the HTTP GET request is being sent to 127.0.0.1:8001, but the gunicorn server is dropping the request on the floor without responding.

Can you share your `/opt/netbox/gunicorn.py` and your `/etc/systemd/system/netbox.service`? Also, is there anything else that you've changed from defaults in your configuration.py (e.g. authentication?)

```
diff -u /opt/netbox/netbox/netbox/configuration{_example,}.py
```

**Reply by Gmello1** [reply]
I was rechecking the main config file, i changed the SSL field in the REDIS config to run without SSL back when i was testing without the certificates, and now that i'm using the correct certificates, i did not changed the field to 'False" again. Well, its working now, thanks for your patience!
